module-4/backend/week_three.md

```diff
@@ -5,15 +5,33 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. At a high level, what is Node?
+  * Node is a backend runtime environment for JavaScript built using Chrome's V8 engine.
+  
 2. What is Express? What is Express similar to in the Ruby world?
+  * Express is a web framework for JavaScript. Express would be similar to Rails except Rails is much larger and more opinionated. Express would be closer to Sinatra but Sinatra is a DSL not a web framework. So I guess Express is most similar to Padrino.
+
 3. How do we setup a route when creating an API with Node and Express? Please provide a code snippet.
+  * Verb, then path, then ideally a controller method. Something like `app.get('/home', homeController.getAllData)`
+  
 4. What do we use Knex for?
+  * Knex is a SQL query builder for JavaScript. It allows us to query our database to return relevant information.
+  
 5. How could you organize your code to follow the MVC design pattern in your Quantified Self project?
+  * Using Express we limited database queries to the model layer, building functions that would then be called by the controller, which would in turn serialize the information for the views. The routes also lived in a separate layer, only calling on controller methods. So the MVC pattern was similar to a Rails app, expect that we didn't have a separte views layer.
```

Comment:
in this case, we don't have a separate view layer, but we could if we wanted to!